ui/packages/ui/webpack.common.js

```diff
@@ -6,6 +6,7 @@ const { dependencies } = require("./package.json");
 
 // Try the environment variable, otherwise use root
 const ASSET_PATH = process.env.ASSET_PATH || '/';
+const COMMIT_HASH = require('child_process').execSync('git rev-parse --short HEAD').toString().trim();
```

Comment:
Probably a real noob question: but this runs at _build-time_, right? I.e. there's no assumption that this is run from within the source repo, correct?

Reply:
Not a noob question at all! I believe that is how this works, it should get evaluated when `start` or `build` are invoked. according to the docs 

> The DefinePlugin allows you to create global constants which can be configured at compile time.
https://webpack.js.org/plugins/define-plugin/#root